cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -48,9 +48,10 @@ import (
 )
 
 var (
-	version = flag.Bool("version", false, "Print the version of the proxy and exit")
-	verbose = flag.Bool("verbose", true, "If false, verbose output such as information about when connections are created/closed without error are suppressed")
-	quiet   = flag.Bool("quiet", false, "Disable log messages")
+	version        = flag.Bool("version", false, "Print the version of the proxy and exit")
+	verbose        = flag.Bool("verbose", true, "If false, verbose output such as information about when connections are created/closed without error are suppressed")
+	quiet          = flag.Bool("quiet", false, "Disable log messages")
+	logDebugStdout = flag.Bool("log_debug_stdout", false, "If true, log messages that are not errors will outout to stdout instead of stderr")
```

Comment:
Sorry to not catch this earlier, but it says "outout" instead of output.

Reply:
I need to turn spell check on. Pushed a fix :+1:

Reply:
Thanks, and sorry for the nitpicking ;)